lib/custom_a11y_rules.js

```diff
@@ -0,0 +1,151 @@
+        'nav-aria-label-present',
+        'nav-aria-label-value'
+      ],
+      'any': [],
+      'none': [],
+      'metadata': {
+        'description': 'Ensures <nav> elements have valid aria-label attribute.',
+        'help': '<nav> elements MUST have an aria-label, and the aria-label MUST NOT have ' +
+                '\'navigation\' or \'nav\' as part of the value.',
+        'helpUrl': 'https://openedx.atlassian.net/wiki/display/A11Y/edX+Specific+Accessibility+Tests'
+      }
+    },
+    {
+      'id': 'link-href',
+      'selector': 'a',
+      'enabled': true,
+      'tags': [ 'edx-custom' ],
+      'all': [
+        'link-href',
+        'link-href-internal'
+      ],
+      'any': [],
+      'none': [],
+      'metadata': {
+        'description': 'Ensures <a> elements have acceptable href values',
+        'help': 'all links must have an href whose value MUST NOT be empty or "#"',
+        'helpUrl': 'https://openedx.atlassian.net/wiki/display/A11Y/edX+Specific+Accessibility+Tests'
+      }
+    },
+    {
+      'id': 'skip-link',
+      'selector': 'a[href]',
+      'pageLevel': true,
+      'enabled': true,
+      'tags': ['edx-custom'],
+      'all': [],
+      'any': ['skip-link'],
+      'none': [],
+      'metadata': {
+          'description': 'Ensures the first link on the page is a skip link',
+          'help': 'The page should have a skip link as its first link',
+          'helpUrl': 'https://dequeuniversity.com/rules/axe/1.1/skip-link'
+      }
+    },
+    {
+      'id': 'icon-aria-hidden',
+      'selector': 'i.icon, i.fa, span.icon, span.fa',
+      'enabled': true,
+      'excludeHidden': false,
+      'tags': ['edx-custom'],
+      'all': [
+        'icon-aria-hidden'
+      ],
+      'any': [],
+      'none': [],
+      'metadata': {
+          'description': 'Ensures all icon-font elements are aria-hidden',
+          'help': 'All <i> elements should have aria-hidden="true"',
+          'helpUrl': 'https://openedx.atlassian.net/wiki/display/A11Y/edX+Specific+Accessibility+Tests'
+      }
+    }
+  ],
+  'checks': [
+    {
+      'id': 'nav-aria-label-present',
+      'metadata': {
+        'impact': 'critical',
+        'messages': {
+          'pass': 'all <nav> elements MUST have an aria-label or aria-labelledby attribute',
+          'fail': 'all <nav> elements MUST have aria-label or aria-labelledby attribute'
+        }
+      },
+      evaluate: function(node, options) {
+        return axe.commons.aria.label(node) !== null;
+      },
+    },
+    {
+      'id': 'nav-aria-label-value',
+      'metadata': {
+        'impact': 'critical',
+        'messages': {
+          'pass': 'aria-label values on <nav> elements MUST NOT have "navigation" or "nav" as part of their value',
+          'fail': 'aria-label values on <nav> elements MUST NOT have "navigation" or "nav" as part of their value'
+        }
+      },
+      evaluate: function(node, options) {
+        label = axe.commons.aria.label(node) || '';
```

Comment:
Declaring these with `var` will prevent polluting the global namespace.  Typically, this is done with one var, e.g.:

```
var label = axe.commons.aria.label(node) || '',
      words = ...;
```